frontend/lib/pages/data-driven-onboarding.tsx

```diff
@@ -139,7 +139,7 @@ function ActionCardIndicators(props: Pick<ActionCardProps, 'indicators'|'fallbac
 
   return <>
     {indicators.map((indicator, i) => (
-      <p key={i} className="subtitle is-spaced">{indicator}</p>
```

Comment:
I also changed these guys to be `div`'s, not `p`'s as we are now nesting `p`'s inside them. Any reason they were originally `p`s?